keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -957,55 +961,84 @@ complete their service.
+of the objects exhibited in the
+[Example Configuration](#example-configuration) section above.
+
+Whenever a kube-apiserver starts up, it checks whether there is a
+PriorityLevelConfiguration object with `Spec.Exempt==true`.  If not
+then the apiserver attempts to create the predefined configuration objects, the
+exempt PriorityLevelConfiguration last.  In this case: for each predefined
+object, the apiserver will create that object if there is not already
+an object with the same GroupKind and name; note that mere existence
+is enough; the apiserver does not insist that the spec matches its
+predefined content.
+
+Administrators can choose to persist a partially or completely
+different set of configuration API objects than the predefined set ---
+so long as the persisted set includes a PriorityLevelConfiguration
+object with `Spec.Exempt==true`.  (Remember that the behavior of the
+apiserver before it has had a chance to read the persisted
+configuration API objects is a different matter.)  Any configuration
+object can be created, updated, and deleted.
+
+#### API Server Initial Behavior
+
+As it starts up, an apiserver has to start handling requests before
+the filter discussed here has had a chance to read the persisted
+configuration API objects.  Indeed, that reading will have to be done
+through the very same filter (at least in the first apiserver in a
+cluster)!  The initial request handling also applies while an
+apiserver is persisting the default configuration API objects for the
+cluster.
+
+When the apiserver first starts up, it behaves as if the
+configuration consists of just (1) the predefined
+PriorityLevelConfiguration objects named `exempt` and `default`, and
+(2) the predefined FlowSchema objects named `exempt` and `default`.
+
+An apiserver maintains this initial behavior until it has had a
+chance to read all the persisted configuration API objects.
+
+After the initial behavior period is over, regular behavior begins.
+
+#### Default Request Handling
+
+It is possible that the administrators can modify the configuration
+API objects to create a situation where some requests will not match
+any FlowSchema.  This is also the situation before the cluster has
+been given its initial load of configuration API objects.  Once the
+initial behavior period is over, any request that does not match a
+FlowSchema object is handled as follows.
+
+If an unmatched request has `system:masters` among its user groups
+then the request is considered exempt and is assigned to one of the
+exempt priority levels.  If no PriorityLevelConfiguration object has
+`Spec.Exempt==true` then the apiserver acts as if there is an
+additional PriorityLevelConfiguration object with `Spec.Exempt==true`;
+for the rest of the `Spec` the apiserver will use either (a) what it
+most recently saw associated with `Exempt==true` or (b) the
+configuration as shown for the `exempt` PriorityLevelConfiguration
+above.
```

Comment:
This is actually pointless, an exempt PriorityLevelConfiguration has nothing else that can be set.

Reply:
true for the latest API model